[ADD] Quality: add Create quality alerts (#3715)
odoo/documentation: content/applications/inventory_and_mrp/manufacturing/quality_control/quality_alerts.rst

=====================
Create quality alerts
=====================

:doc:`Configuring quality control points <quality_control_points>` is a great way to ensure that
quality checks are performed at routine stages during specific operations. However, quality issues

> can often appear outside of these scheduled checks. Using Odoo Quality, users can create quality

Reviewer: Italicize Odoo Quality because it is an app name

alerts for issues that are not detected by automated processes.

Create quality alerts from the quality module
=============================================

> In some situations, it is necessary to manually create quality alerts within the Quality module.

Reviewer: Italicize Quality

.. example::
    A helpdesk user who is notified of a product defect by a customer ticket can create an alert that

>     brings the issue to the attention of the relevant quality team.

Reviewer: This text is indented by one space too many, all text within an admotion block should align with the first letter of the admonition type: `e`xample

> To create a new quality alert, start from the Quality module and select :menuselection:`Quality

Reviewer: add :menuselection:Quality

  applicable
- :guilabel:`Lot`: the lot number assigned to the product
- :guilabel:`Work Center`: the work center where the quality issue originated
- :guilabel:`Picking`: the picking operation during which the quality issue originated
- :guilabel:`Team`: the quality team that will be notified by the quality alert
- :guilabel:`Responsible`: the individual responsible for managing the quality alert
- :guilabel:`Tags`: classify the quality alert based on user-created tags
- :guilabel:`Root Cause`: the cause of the quality issue, if known
- :guilabel:`Priority`: assign a priority between one and three stars to ensure more
  urgent issues are prioritized

The tabs at the bottom of the form can be used to provide additional information to quality teams:

- :guilabel:`Description`: provide additional details about the quality issue
- :guilabel:`Corrective Actions`: the method for fixing affected products
- :guilabel:`Preventive Actions`: procedures for preventing the issue from occurring in
  the future
- :guilabel:`Miscellaneous`: the product vendor (if applicable), the company that produces
  the product, and the date assigned

> .. image:: quality_alerts/quality_alert_form.png

Reviewer: All media files should be hyphenated, filenames should not contain underscores. See: https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#media-files

    :align: center
    :alt: An example of a completed quality alert form.

Create quality alerts during the manufacturing process
======================================================

Odoo enables manufacturing employees to create quality alerts within a work order without
accessing the Quality module. From the work order tablet view, click the :guilabel:`Menu` button in

> the top left corner and select :guilabel:`Quality Alert`.

Reviewer: the lines in this paragraph can be broken later

> .. image:: quality_alerts/work_order_tablet_view_menu_button.png

Reviewer: underscores in image filename

    :align: center
    :alt: Access the work order menu.

The quality alert form can then be filled out as detailed in the previous section. After saving the

> form, a new alert will appear on the :guilabel:`Quality Alerts` page of the Quality module.

Reviewer: Italicize Quality

Manage existing quality alerts
==============================

By default, quality alerts are organized in a Kanban board view. The stages of the Kanban board are

> fully configurable and alerts can be moved from one stage to the next by dragging and dropping or

Reviewer: lowercase kanban

=====================
Create quality alerts
=====================

> :doc:`Configuring quality control points <quality_control_points>` is a great way to ensure that
> quality checks are performed at routine stages during specific operations. However, quality issues
> can often appear outside of these scheduled checks. Using *Odoo Quality*, users can create quality
> alerts for issues that are not detected by automated processes.

Reviewer 2: Think this doc link serves better as a `seealso`.
As well (separate), maybe we should rename the other doc's H1 to say `Add quality control points` or something like that...a little more specificity can help differentiate these two docs some more.
```suggestion
Configuring quality control points is a great way to ensure that quality checks are performed at
routine stages during specific operations. However, quality issues can often appear outside of these
scheduled checks. Using Odoo *Quality*, users can create quality alerts for issues that are not
detected by automated processes.

.. seealso::
   :doc:`Add quality control points <quality_control_points>`
```

> Create quality alerts from the quality module
> =============================================

Reviewer 2: Seems redundant to the h1.
Here's my suggestion that differentiates the section and still reflects what's being discussed.
```suggestion
Find and fill out the quality alerts form
=========================================
```

- :guilabel:`Work Center`: the work center where the quality issue originated
- :guilabel:`Picking`: the picking operation during which the quality issue originated
- :guilabel:`Team`: the quality team that will be notified by the quality alert
- :guilabel:`Responsible`: the individual responsible for managing the quality alert
- :guilabel:`Tags`: classify the quality alert based on user-created tags
- :guilabel:`Root Cause`: the cause of the quality issue, if known
- :guilabel:`Priority`: assign a priority between one and three stars to ensure more
  urgent issues are prioritized

The tabs at the bottom of the form can be used to provide additional information to quality teams:

- :guilabel:`Description`: provide additional details about the quality issue
- :guilabel:`Corrective Actions`: the method for fixing affected products
- :guilabel:`Preventive Actions`: procedures for preventing the issue from occurring in
  the future
- :guilabel:`Miscellaneous`: the product vendor (if applicable), the company that produces
  the product, and the date assigned

.. image:: quality_alerts/quality-alert-form.png
   :align: center
   :alt: An example of a completed quality alert form.

> Create quality alerts during the manufacturing process
> ======================================================

Reviewer 2: Nitpick:
Can choose a different word than "Create" just to add some variation in the headings 😉 
```suggestion
Add quality alerts during the manufacturing process
===================================================
```

Odoo enables manufacturing employees to create quality alerts within a work order without accessing

> the *Quality* module. From the work order tablet view, click the :guilabel:`Menu` button in the top
> left corner and select :guilabel:`Quality Alert`.

Reviewer 2: **Menu** doesn't quite capture the actual guilabel. Can use the hamburger symbol here and contextualize it in the words following.
```suggestion
the *Quality* module. From the work order tablet view, click the :guilabel:` ☰ ` hamburger menu
icon in the top left corner and select :guilabel:`Quality Alert`.
```

.. image:: quality_alerts/work-order-tablet-view-menu-button.png
   :align: center
   :alt: Access the work order menu.

The quality alert form can then be filled out as detailed in the previous section. After saving the

> form, a new alert will appear on the :guilabel:`Quality Alerts` page of the *Quality* module.

Reviewer 2: Trying to force clearer tag use here...
Here's what I came up with
```suggestion
form, a new alert will appear on the :guilabel:`Quality Alerts` dashboard that can be found through
the :menuselection:`Quality --> Quality Control` menu.
```

Manage existing quality alerts
==============================

By default, quality alerts are organized in a kanban board view. The stages of the kanban board are

> fully configurable and alerts can be moved from one stage to the next by dragging and dropping or

Reviewer 2: Missing kanban stage settings walk-through when you hover over a stage and click the gear icon, which are unique to this dashboard.

Can add now or on another PR as a quick fix. Up to you 🙂

from within each alert. Additional options are available for viewing alerts, including graph,
calendar, and pivot table views.

> .. tip::
>    Users can filter alerts based on criteria like date assigned or date closed. Alerts can also be
>    grouped by quality team, root cause, or other details.

Reviewer 2: Favor active instruction here instead of having "users" as the subject. Added a little bit more specificity too.

The parameters like _Date Assigned_, _Date Closed_, etc.  are borderline `guilabels` since they're choices on the **Filters** menu, but I _think_ it's best to leave them since it's mostly a hypothetical we're entertaining here and not an active instruction. Alternatively we could capitalize as they're reflected on the UI and italicize. Up to you 🙂 
```suggestion
.. tip::
   Filter alerts based on diverse criteria like date assigned or date closed. Alerts can also be
   grouped by quality team, root cause, or other details.
```
